src/controllers/eventsController.ts

```diff
@@ -0,0 +1,76 @@
+import eventsService from '../services/eventsService';
+
+import { validationResult } from 'express-validator';
+import ApiError from '../errors/ApiError';
+
+import { NextFunction, Request, Response } from 'express';
+import EventInfo from '../types/EventInfo';
+
+class EventsController {
+  public async getEventsList(
+    req: Request,
+    res: Response,
+    next: NextFunction,
+  ): Promise<Array<EventInfo> | void> {
```

Comment:
It's okay to use Array<EventInfo>
But if u have another generic better to use Promise<EventInfo[]>